README.md

```diff
@@ -33,7 +33,7 @@ At this point the `ProvisionNetworking` policy is attached to the
 
 | Name | Version |
 |------|---------|
-| terraform | ~> 0.12.0 |
+| terraform | ~> 0.13.0 |
 | aws | ~> 3.38 |
 
 ## Providers ##
```

Comment:
Would you update the rest of the `terraform-docs` generated output in this README? I noticed that the `Modules` section is not using the latest formatting style.

Reply:
Thanks for catching that - see https://github.com/cisagov/cool-sharedservices-networking/pull/53/commits/003993958f8c6bc7f87cb3504c0bdb96c3fe5ef9.  I've been trying to check these as I go, but clearly I have missed some.